src/main/scala/se/lu/nateko/cp/data/formats/delimitedheadercsv/DelimitedHeaderCsvStreams.scala

```diff
@@ -10,25 +10,46 @@ import java.time.Instant
 import se.lu.nateko.cp.meta.core.data.IngestionMetadataExtract
 import akka.stream.scaladsl.Keep
 import java.time.temporal.ChronoUnit
-import java.time.{ Instant, LocalDateTime, ZoneOffset }
+import java.time.{ Instant, LocalDateTime, LocalDate, LocalTime, ZoneOffset }
 import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeFormatterBuilder
+import java.time.temporal.ChronoField
 
 object SitesDelimitedHeaderCsvStreams extends StandardCsvStreams {
```

Comment:
Should be converted to a class accepting `ColumnsMeta` instance as constructor argument. This way, no change of signatures of `StandardCsvStreams` is necessary.